IBPSA/Fluid/Chillers/ModularReversible/Data/TableData2D/EN14511/SingleSplitRXM20R.mo

```diff
@@ -0,0 +1,50 @@
+within IBPSA.Fluid.Chillers.ModularReversible.Data.TableData2D.EN14511;
+record SingleSplitRXM20R "Daikin_FTXM20R_RXM20R cooling mode"
+  extends IBPSA.Fluid.Chillers.ModularReversible.Data.TableData2D.Generic(
+    dpEva_nominal=0,
```

Comment:
Enter `dpEva_nominal` and `dpCon_nominal` from the manufacturer data sheet.
If not available, set them to zero in the example; otherwise users will simulate models with these data assuming they capture the pressure drop.

Reply:
@mwetter : I entered the values as available in the datasheet. Some of the mentioned components, e.g. this one, does not need `dp_nominal`, as the movers power consumption is typically included in the measured data in air-source/sink devices.

Should I remove the default values in the examples only? Or the default binding of `dp_nominal` in models such as `IBPSA.Fluid.HeatPumps.ModularReversible.TableData2D`? In my opinion the user has to check the table data in any way to understand the data quality used for the model. Thus, I would like to keep the default binding.

Reply:
I didn't notice that this is a wall mounted HP that won't need ducting/piping. In this case let's leave to default value of 0.